port/win32/omrsysinfo.c

```diff
@@ -90,6 +91,40 @@ omrsysinfo_get_CPU_architecture(struct OMRPortLibrary *portLibrary)
 #endif
 }
 
+intptr_t
+omrsysinfo_get_processor_description(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc)
+{
+	intptr_t rc = -1;
+	Trc_PRT_sysinfo_get_processor_description_Entered(desc);
+	
+	if (NULL != desc) {
+		memset(desc, 0, sizeof(OMRProcessorDesc));
+		rc = getX86Description(portLibrary, desc);
+	}
+	
+	Trc_PRT_sysinfo_get_processor_description_Exit(rc);
+	return rc;
+}
+
+BOOLEAN
+omrsysinfo_processor_has_feature(struct OMRPortLibrary *portLibrary, OMRProcessorDesc *desc, uint32_t feature)
+{
+	BOOLEAN rc = FALSE;
+	Trc_PRT_sysinfo_processor_has_feature_Entered(desc, feature);
+
+	if ((NULL != desc)
+	&& (feature < (OMRPORT_SYSINFO_OS_FEATURES_SIZE * 32))
+	) {
```

Comment:
This formatting is... interesting :-)

Reply:
Sorry, not too sure what happened here

Reply:
Should we also do this for `omrsysinfo_get_processor_description`